docs/protocol/toc.js

```diff
@@ -22,8 +22,9 @@ const toc = [
     link: { type: "doc", id: "protocol/reference/index" },
     items: [
       "protocol/reference/deployed",
-      "protocol/reference/api",
```

Comment:
Could we keep the name of this file and change the contents to be the updated contents in `subgraph-info`? The only reason is because of broken links, and it is probably easier to just keep the name instead of setting up redirects for the name change.